cp-ksql-server/include/etc/confluent/docker/configure

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset \
+    -o errexit \
+    -o verbose \
+    -o xtrace
+
+dub ensure KSQL_BOOTSTRAP_SERVERS
```

Comment:
Question: Are we now advocating the use of `bootstrap.servers` or `ksql.streams.bootstrap.servers` (cf.
https://docs.confluent.io/current/ksql/docs/installation/server-config/config-reference.html#ksql-streams-bootstrap-servers)?

Because the shorter `bootstrap.servers` is not documented in the config reference, even though the *example* in the link above uses the shorter form.

Reply:
I updated the scripts to accept either `KSQL_BOOTSTRAP_SERVERS` or `KSQL_KSQL_STREAMS_BOOTSTRAP_SERVERS`.